src/plugins/presentation_util/public/components/shape.tsx

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
Can all of this Shape stuff that is in presentation_util move to the expression_shape plugin? It doesn't seem like any of it is shared amongst many plugins.

Reply:
I'd like to say that it is used at the `expression_progress` too. If you'll look at this [PR](https://github.com/elastic/kibana/pull/104457), especially at [this file](https://github.com/elastic/kibana/blob/b53806eee379e4fbeb5f2ed84783564d34551579/src/plugins/expression_progress/public/components/shapes/semicircle.tsx), as an example, you'll admit, that this code in the perspective is used by other `expressions`. That's why I've extracted the current component to a `presentation_util`.